bin/ovpn_genconfig

```diff
@@ -104,6 +110,11 @@ while getopts ":r:s:du:cp:DNm:" opt; do
     esac
 done
 
+# if new routes were not defined with -r, use default
+[ ${#TMP_ROUTES[@]} -gt 0 ] && OVPN_ROUTES=("${TMP_ROUTES[@]}")
+
+# if new push directives were not defined with -p, use default
+[ ${#TMP_PUSH[@]} -gt 0 ] && OVPN_PUSH=("${TMP_PUSH[@]}")
```

Comment:
I understand the need for this, but why not implement it like OVPN_ROUTES was previously line 123/134?